.gitlab-ci.yml

```diff
@@ -806,6 +806,9 @@ library:ci-math_classes:
 library:ci-mathcomp:
   extends: .ci-template-flambda
 
+library:ci-mczify:
+  extends: .ci-template-flambda
```

Comment:
If it depends on mathcomp this should be declared here (see eg how vst depends on flocq below, or mathclasses on bignums above)

Reply:
Thanks.